pom.xml

```diff
@@ -1,64 +1,64 @@
-    <repository>
-      <releases>
-        <enabled>true</enabled>
-        <updatePolicy>always</updatePolicy>
-        <checksumPolicy>fail</checksumPolicy>
-      </releases>
-      <snapshots>
-        <enabled>false</enabled>
-      </snapshots>
-      <id>nexus-releases</id>
-      <name>QBiC Releases</name>
-      <url>https://qbic-repo.qbic.uni-tuebingen.de/repository/maven-releases</url>
-    </repository>
-  </repositories>
-  <dependencies>
-    <dependency>
-      <groupId>life.qbic</groupId>
-      <artifactId>openbis-api</artifactId>
-      <version>18.06.2</version>
-    </dependency>
-    <dependency>
-      <groupId>life.qbic</groupId>
-      <artifactId>openbis-core</artifactId>
-      <version>18.06.2</version>
-    </dependency>
-    <!-- https://mvnrepository.com/artifact/com.google.truth/truth -->
-    <dependency>
-      <groupId>com.google.truth</groupId>
-      <artifactId>truth</artifactId>
-      <version>1.0.1</version>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>life.qbic</groupId>
+		<artifactId>parent-pom</artifactId>
+		<version>3.1.1</version>
+	</parent>
+	<artifactId>openbis-client-lib</artifactId>
+	<packaging>jar</packaging>
+	<version>1.4.1</version>
+	<name>openBIS client library</name>
+	<!-- we only need to tell maven where to find our parent pom and other QBiC 
+		dependencies -->
+	<repositories>
+		<repository>
+			<releases>
+				<enabled>false</enabled>
+			</releases>
+			<snapshots>
+				<enabled>true</enabled>
+				<updatePolicy>always</updatePolicy>
+				<checksumPolicy>fail</checksumPolicy>
+			</snapshots>
+			<id>nexus-snapshots</id>
+			<name>QBiC Snapshots</name>
+			<url>https://qbic-repo.qbic.uni-tuebingen.de/repository/maven-snapshots</url>
+		</repository>
+		<repository>
+			<releases>
+				<enabled>true</enabled>
+				<updatePolicy>always</updatePolicy>
+				<checksumPolicy>fail</checksumPolicy>
+			</releases>
+			<snapshots>
+				<enabled>false</enabled>
+			</snapshots>
+			<id>nexus-releases</id>
+			<name>QBiC Releases</name>
+			<url>https://qbic-repo.qbic.uni-tuebingen.de/repository/maven-releases</url>
+		</repository>
+	</repositories>
+	<dependencies>
+		<dependency>
+			<groupId>life.qbic</groupId>
+			<artifactId>openbis-api</artifactId>
+			<version>18.06.2</version>
+		</dependency>
+		<dependency>
+			<groupId>life.qbic</groupId>
+			<artifactId>openbis-core</artifactId>
+			<version>18.06.2</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.google.truth/truth -->
+		<dependency>
+			<groupId>com.google.truth</groupId>
+			<artifactId>truth</artifactId>
+			<version>1.0.1</version>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
 </project>
```

Comment:
was this changed?

Reply:
no, just formatted after merge with previous PR :/

Reply:
we figure that this is due to tabs vs. spaces